src/protocol.ts

```diff
@@ -123,21 +125,47 @@ export class Protocol {
   }
 
   /**
-   * Fetches current collateral supply (Amount of wstETH locked in Raft protocol).
-   * @returns Fetched collateral supply
+   * Fetches current collateral supply for each underlying token.
+   * @returns List of fetched collateral supplies.
```

Comment:
```suggestion
   * @returns Fetched collateral supplies per underlying collateral token.
```

Same should be applied to `fetchDebtSupply`